src/Main.java

```diff
@@ -78,16 +70,46 @@ private static void userInterface(boolean isCorrectCarData) {
 		userInterface(isCorrectCarData);// REKURENCJA - WYWOLUJE SIE PONOWNIE
 										// METODA
 	}
+
+	private static void addEmployeeM() throws IncorrectUserInterfaceDataException {
```

Comment:
please, have a look deeper in this section

Reply:
looks good